tests/foreman/api/test_errata.py

```diff
@@ -17,23 +17,28 @@
 :Upstream: No
 """
 # For ease of use hc refers to host-collection throughout this document
+from datetime import datetime
 from time import sleep
 
 import pytest
 from broker import VMBroker
 from nailgun import entities
 
 from robottelo import constants
+from robottelo import manifests
 from robottelo.api.utils import enable_rhrepo_and_fetchid
 from robottelo.api.utils import promote
+from robottelo.api.utils import upload_manifest
+from robottelo.api.utils import wait_for_tasks
 from robottelo.cli.factory import setup_org_for_a_custom_repo
 from robottelo.cli.factory import setup_org_for_a_rh_repo
+from robottelo.cli.job_invocation import JobInvocation
```

Comment:
Dont we have API entity in nailgun for this ?

Reply:
We do. I was getting some flakiness time out in the polling after the Job from API. As the test is focused upon getting the errata installed irrespective of how job is invoked, I choose to use the CLI instead.